File: 2-second-year-2022-2023/Programming-languages-and-laboratory/Python/Exercises/31_compito_java_20190705/31_GestioneVeicoli_python/Auto.py
from Veicolo import Veicolo


class Auto(Veicolo):
    def __init__(self, codice, tipo, targa, modello, marca, bagagliaio, cilindrata, categoria, costo_giornaliero):
        super().__init__(codice, tipo, targa, modello, marca, costo_giornaliero)
        self._bagagliaio = bagagliaio
        self._cilindrata = cilindrata
        self._categoria = categoria

    def __str__(self):
        return "automobile" + "\t" + super().__str__() + "\t" + str(self._bagagliaio) + "\t" + str(self._cilindrata) + "\t" + self._categoria + "\t-\t-\t-"

File: II - Linguaggi di programmazione e laboratorio/Java/Exercises/40_compito_java_2020_06_22/Auto.java
import java.util.*;

public class Auto extends Veicolo {

    private int cilindrata;
    private double bagagliaio;

    public Auto(int codice, String targa, String modello,
                String marca, double costo, int cilindrata, double bagagliaio) {
        super(codice, targa, modello,
                marca, costo);
        this.cilindrata = cilindrata;
        this.bagagliaio = bagagliaio;
    }

    public String toString() {
        return "automobile\t" + super.toString() + cilindrata + "\t" + bagagliaio + "-\t-\t-";
    }

}